test.md

```diff
@@ -0,0 +1 @@
+this text should be ignored by git
```

Comment:
- This file should be ignored by git

Reply:
Hello I have tried putting the test.md file in the .gitignore file but it seems like the test.md file is'nt being ignored I dont understand. What do I do?

Reply:
Please explain do i need to change the text?

Reply:
> Please explain do i need to change the text?

No you need just to add this file to gitignore

Reply:
I have done that already.